examples/react/atoms/src/index.tsx

```
@@ -0,0 +1,62 @@
+import ReactDOM from 'react-dom/client'
+import {
+  type Atom,
```

ESLint

[error] Prefer using a top-level type-only import instead of inline type specifiers.

(import/consistent-type-specifier-style)